Formula/nfdump.rb

```diff
@@ -12,9 +13,11 @@ class Nfdump < Formula
     sha256 "09a51a71d30f8045075f11f8511a4ec0d6bfd6f9d3cbc3bbb2927c911fce1ce1" => :mavericks
   end
 
+  depends_on "automake" => :build
+
   def install
     system "./configure", "--prefix=#{prefix}", "--enable-readpcap"
-    system "make", "install"
+    system "make", "-j1", "install"
```

Comment:
Don't hardcode `-j1`. If it can't build parallelised do:
```ruby
ENV.deparallelize { system "make", "install" }
```
That would ideally be reported upstream as well. Things should build in parallel.